app/controllers/group_assignment_invitations_controller.rb

```diff
@@ -69,6 +49,14 @@ def required_scopes
     GitHubClassroom::Scopes::GROUP_ASSIGNMENT_STUDENT
   end
 
+  def group_params
+    params
+      .require(:group)
+      .permit(:id, :title)
```

Comment:
This method isn't new, it was just moved to a different spot in the file for organization purposes.